doc/python/conf.py.in

```diff
@@ -0,0 +1,189 @@
+# Copyright (c) 2016, EPFL/Blue Brain Project
```

Comment:
This is a separate thing. Pydoxine just generates the docstrings to include in the wrapping, this is the Sphinx config to generate the html module documentation.